pkg/azureutils/azure_disk_utils.go

```diff
@@ -115,12 +116,16 @@ const (
 	// to prevent the pod deletion until clean up is completed
 	ControllerFinalizer              = "disk.csi.azure.com/azuredisk-finalizer"
 	VolumeAttachmentExistsAnnotation = "disk.csi.azure.com/volume-attachment"
+	CleanUpAnnotation                = "disk.csi.azure.com/clean-up"
 	VolumeDetachRequestAnnotation    = "disk.csi.azure.com/volume-detach-request"
 	VolumeDeleteRequestAnnotation    = "disk.csi.azure.com/volume-delete-request"
 	NodeNameLabel                    = "node-name"
 	VolumeNameLabel                  = "volume-name"
 	RoleLabel                        = "requested-role"
 
+	DefaultProvisionTimeout = time.Duration(15) * time.Second
+	DefaultAttachTimeout    = time.Duration(5) * time.Minute
```

Comment:
The only usage I see of this is to set the polling interval for the replica attach operation: [`replica.go#L120`](https://github.com/abhisheksinghbaghel/azuredisk-csi-driver/blob/669992ab3721f782b05a71b73e3c22fd4113e252/pkg/controller/replica.go#L120)